projects/plugins/jetpack/extensions/blocks/recurring-payments/test/fixtures/jetpack__recurring-payments__deprecated-2.serialized.html

```diff
@@ -0,0 +1,5 @@
+<!-- wp:jetpack/recurring-payments -->
+<div class="wp-block-jetpack-recurring-payments"><!-- wp:jetpack/recurring-payments-button {"planId":635} -->
```

Comment:
Plan id should be inside the button?

Reply:
Sorry, I don't follow, I think it should be on the jetpack/recurring-payments-button as it is here.

Reply:
My bad, I got the order of things mixed up, you can ignore all the comments like this.